web/modules/custom/portland/portland.module

```diff
@@ -915,6 +915,13 @@ function _notify_reviewer($entity, $entity_event_type)
         'draft' => 'Draft',
         'archived' => 'Unpublished/Archived',
         'published' => 'Published',
+        'unpublished' => 'Unpublished',
```

Comment:
If this is user facing, will need to remove "Unpublished" from the display message for "archived" to avoid confusion.

Reply:
Good catch. The "archived" machine name is associated with the "Unpublished" label in the default workflow. The label of "Unpublished/archived" is legacy. 😄